tslint.json

```diff
@@ -14,7 +14,16 @@
 			2
 		],
 		"no-conditional-assignment": false,
-		"arrow-parens": false
+		"arrow-parens": false,
+		"no-redundant-jsdoc": true,
+		"completed-docs": [
+		  true,
+		  {
+		    "functions": {"visibilities": ["exported"]},
+		    "methods": {"privacies": ["public", "protected"]}
+		  }
+		],
+		"file-name-casing": [true, "camel-case"]

```

Comment:
Forces to have .ts and .tsx files in camelCases which is pretty much the convention that we have now.